src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/RcwAroundCcwTests.cs

```diff
@@ -293,6 +293,7 @@ public void Set(int[][] array, int[] widths, int length)
                 _data = array;
                 _widths = widths;
             }
+            public void asds([MarshalUsing(ConstantElementCount = 10)] ref int[][][][] array) { }
```

Comment:
Is this used anywhere? Would it be more useful to move this to the unit tests instead of having it in the integration tests?